test/fw/ptl/lib/ptl_sched.py

```diff
@@ -802,6 +802,18 @@ def set_dedicated_time_file(self, filename):
                          str(filename))
         self.dedicated_time_file = filename
 
+    def revert_fairshare(self):
+        """
+        Helper method to revert scheduler's fairshare tree.
+        """
+        cmd = [os.path.join(self.pbs_conf['PBS_EXEC'], 'sbin', 'pbsfs'), '-e']
+        if self.sc_name is not 'default':
+            cmd += ['-I', self.sc_name]
+        self.du.run_cmd(cmd=cmd, runas=self.user)
+        self.parse_sched_config()
```

Comment:
I see this was there before, but I don't know why we parse the sched config here.  If it reads in the fairshare tree and resource group file, we immediately revert them below.  Otherwise rereading the sched_config doesn't have much to do with reverting fairshare.  It does need to happen, but not necessarily here.  Maybe it was just in the block of code in revert_to_defaults().  Now that you are moving it in to a new function, I don't think it belongs here.

Reply:
I'll move the call to parse config file back to revert_to_default().